Lib/test/test_typing.py

```diff
@@ -523,6 +527,10 @@ def test_ellipsis_in_generic(self):
         # Shouldn't crash; see https://github.com/python/typing/issues/259
         typing.List[Callable[..., str]]
 
+    def test_or_and_ror(self):
+        Callable = self.Callable
+        self.assertEqual(Callable | bool, Union[Callable, bool])
+        self.assertEqual(bool | Callable, Union[bool, Callable])
```

Comment:
See the above comment for context. 
```suggestion
        Callable = self.Callable
        self.assertEqual(Callable | Tuple, Union[Callable, Tuple])
        self.assertEqual(Tuple | Callable, Union[Tuple, Callable])
```